org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/commons/CodeActionFactory.java

```diff
@@ -169,4 +170,22 @@ public static CodeAction createFile(String title, String docURI, String content,
 
 		return codeAction;
 	}
+
+	/**
+	 * Makes a CodeAction to call a command from the available server commands.
+	 *
+	 * @param title       The displayed name of the CodeAction
+	 * @param documentURI The document URI of the document the command is called on
+	 * @param commandId   The id of the given command to add as CodeAction
+	 * @param diagnostic  The diagnostic that this CodeAction will fix
+	 */
+	public static CodeAction createCommand(String title, String documentURI, String commandId, Diagnostic diagnostic) {

```

Comment:
I think it would be better to make this method more generic, so that it could be reused in the future if need to make another CodeAction that calls a command. For instance, I think this should:
 * Accept an array or a `List` of `Object`s to pass to the command instead of `documentURI`
 * Use the title for the title of the Command as well, instead of a fixed string.
 * Name the `Command` object something less specific

Reply:
The maini idea with this fracture is that it should be generic and not to have custom code.
Indeed i would like to provide a jar library with this  class and any lsp language server  could use it.